source/GameAction.cpp

```diff
@@ -182,6 +182,29 @@ void GameAction::LoadSingle(const DataNode &child)
 		else
 			child.PrintTrace("Error: Skipping invalid \"fine\" with non-positive value:");
 	}
+	else if(key == "mortgage")
+	{
+		GameAction::Mortgage mortgage;
+		for(const DataNode &grand : child)
+		{
+			const string &grandKey = grand.Token(0);
+			bool grandHasValue = (grand.Size() < 2);
+			if(grandKey == "use credit score")
+				mortgage.useCreditScore = true;
```

Comment:
I'm a bit confused with this attribute existing
If it exists the interest attribute is ignored, so isnt this just a lack of interest?

If we do keep this behavior we should add a warning when usecreditscore is true and you try to define an interest, but cant we just have the lack of specified interest mean this by default?